testing/cli/test_cli_sct_label_vertebrae.py

```diff
@@ -51,6 +51,45 @@ def test_sct_label_vertebrae_high_value_warning(caplog, tmp_path):
     assert 'Disc value not included in template.' in caplog.text
 
 
+def test_sct_label_vertebrae_initial_disc_no_inferior(caplog, tmp_path):
```

Comment:
I really love the in-depth descriptions you put into your commits. For example, the description in https://github.com/spinalcordtoolbox/spinalcordtoolbox/commit/a442e1291c060600bed126aeef617737c0296cd6 really aided me when trying to trace the behavior of `sct_label_vertebrae`.

I'm wondering... would you be able to adapt these descriptions into docstrings for the tests? That way, we have a loose description of the script's behavior for each corner case test (i.e. -1, 0, 19, 20, 21) attached to the tests themselves.